src/Utils.cc

```diff
@@ -756,14 +756,14 @@ std::pair<K*, V*> radix_sort_parallel(
 #ifdef _MSC_VER
   const size_t array_size = (size_t)RDX_HIST_SIZE * maxthreads;
   // fixes MSVC error C2131
-  int* const histogram = static_cast<int*>(
-      fbgemm::fbgemmAlignedAlloc(64, (array_size) * sizeof(int)));
-  int* const histogram_ps = static_cast<int*>(
-      fbgemm::fbgemmAlignedAlloc(64, (array_size + 1) * sizeof(int)));
+  const auto histogram = static_cast<int64_t*>(
```

Comment:
For pointer types, we should probably use `auto*` instead of plain `auto` for readability.

Reply:
@q10 Yeah, good idea, I replaced `const auto` with `auto* const`.